Simplified/Book/UI/NYPLBookDetailView.m

```diff
@@ -147,6 +147,30 @@ - (instancetype)initWithBook:(NYPLBook *const)book
   return self;
 }
 
+- (void)traitCollectionDidChange:(UITraitCollection *)previousTraitCollection {
+  [super traitCollectionDidChange:previousTraitCollection];
+  
+  if (@available(iOS 12.0, *)) {
+    if (UIScreen.mainScreen.traitCollection.userInterfaceStyle != previousTraitCollection.userInterfaceStyle) {
+      [self updateColors];
+    }
+  }
+}
+
+- (void)updateColors
+{
+  UIBlurEffectStyle style = UIBlurEffectStyleLight;
+  if (@available(iOS 12.0, *)) {
+    if (UIScreen.mainScreen.traitCollection.userInterfaceStyle == UIUserInterfaceStyleDark) {
+      style = UIBlurEffectStyleDark;
+    }
+  }
+  UIVisualEffect *blurEffect = [UIBlurEffect effectWithStyle:style];
```

Comment:
is this blur effect something we already had? It seems like we are adding it for light and dark mode.